CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2011-2022 Greenbone Networks GmbH
+# Copyright (C) 2011-2022 Greenbone AG
```

Comment:
I think in those cases it must be either adjusted to 2023 or be left at GmbH since 2022 it was GmbH while the AG is the legal successor so I'd guess it would be fine in the cases that weren't changed?

Reply:
will be ignored